models/tokenlon/ethereum/tokenlon_v5_ethereum_rfq_v2_trades.sql

```diff
@@ -0,0 +1,109 @@
+{{ config(
+    schema = 'tokenlon_v5_ethereum',
+    alias = 'rfq_v2_trades',
+    
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "tokenlon",
+                                    \'["keen"]\') }}'
+    )
+}} 
+{% set blockchain = 'ethereum' %}
+{% set project = 'tokenlon' %}
+{% set project_version = '5' %}
+-- block
+{% set project_start_date = "timestamp '2023-09-12'" %}
+
+WITH dexs AS (
+    SELECT 
+        evt_block_time        AS block_time,
+        "user"                AS taker,
```

Comment:
Is `"` needed for these fields? Or is it logically equivalent without `"`?
```suggestion
        user                AS taker,
```